crates/typst-cli/src/fonts.rs

```diff
@@ -1,21 +1,28 @@
-use typst::text::FontVariant;
-use typst_kit::fonts::Fonts;
-
 use crate::args::FontsCommand;
+use ecow::eco_format;
+use std::io::{self, Write};
+use typst::{diag::StrResult, text::FontVariant};
+use typst_kit::fonts::Fonts;
 
 /// Execute a font listing command.
-pub fn fonts(command: &FontsCommand) {
+pub fn fonts(command: &FontsCommand) -> StrResult<()> {
     let fonts = Fonts::searcher()
         .include_system_fonts(!command.font_args.ignore_system_fonts)
         .search_with(&command.font_args.font_paths);
-
+    let mut out = io::stdout();
```

Comment:
I think we can use `std::io::stdout()` instead of importing `io` just for this (seems to be how it's done in compile.rs).